packages/plugins/i18n/server/src/register.ts

```diff
@@ -70,6 +70,8 @@ const extendContentTypes = (strapi: Core.Strapi) => {
       private: false,
       configurable: false,
       visible: false,
+      unstable_virtual: true,
+      useJoinColumn: false,
```

Comment:
you can remove the useJoinColumn too now

Reply:
oops, that was a last minute typo trying to add back useJoinTable specifically to denote that we wouldn't want to use a join table even if it weren't virtual. But I'll just remove it completely.